Source/WebCore/testing/MockGamepadProvider.cpp

```diff
@@ -149,6 +149,17 @@ void MockGamepadProvider::gamepadInputActivity()
     });
 }
 
+void MockGamepadProvider::clearMockGamepads()
+{
+    // disconnect any remaining connected gamepads
```

Comment:
Comments in WebKit are sentences. They need to start with a capital letter and end with a period.